x-pack/plugins/snapshot_restore/common/constants.ts

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export const PLUGIN = {
+  ID: 'snapshot_restore',
+  NAME: 'Snapshot and Restore',
```

Comment:
This one was tricky since I'm trying to avoid calling `i18n` outside of `shim.ts`. In the end, I changed this to be a function, [getI18nName()](https://github.com/elastic/kibana/pull/32276/files#diff-8c3b6d3d1d873dabdcda1e4900c203b7R14), which is [called when the plugin is initialized](https://github.com/elastic/kibana/pull/32276/files#diff-8d167ecd50c7178cf6b990fa036e2c0aR32).

Also, this is called in the legacy world rather than `plugin.ts` to avoid the plugin knowing about `server`, which is needed by `registerLicenseChecker`.

@epixa, any thoughts on accessing `i18n`, or other core/plugin modules from constant files like this case?